WordPress/Resources/AppImages.xcassets/blank.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I'm not sure we need this file any more – seems like you create the navigation bar shadow image in code now in `PostPostViewController`? I'd prefer creating it in code to using an image.